src/com/oltpbenchmark/LatencyRecord.java

```diff
@@ -29,17 +29,15 @@
    * Contains (start time, latency, transactionType, workerid, phaseid) pentiplets
    * in microsecond form.
    */
-  private final ArrayList<Sample[]> values = new ArrayList<Sample[]>();
+  private final ArrayList<Sample[]> values = new ArrayList<>();
   private int nextIndex;
 
   public LatencyRecord(long startNs) {
     assert startNs > 0;
     allocateChunk();
   }
 
-    public void addLatency(int transType, long startNs, long endNs,
-                           long operationStartNs, long operationEndNs,
-                           int workerId, int phaseId) {
+  public void addLatency(int transType, long startNs, long endNs, long operationStartNs, long operationEndNs) {
```

Comment:
we need the operation start and end as well as the start and end iwth the wait times..
You can skip on the phase and the worker ids

Reply:
what's the suggestion here?

Reply:
current signature is sufficient as discussed offline